src/View/Helper.php

```diff
@@ -95,7 +95,7 @@ public function __construct(View $view, array $config = [])
      *
      * @param string $method Method to invoke
      * @param array $params Array of params for the method.
-     * @return void
+     * @return mixed|void
```

Comment:
Should then rather be mixed|null right?

Reply:
Nah, the method doesn't return anything so psalm expects `void`.

Reply:
Why is mixed then necessary? It shouldnt. Having it both not return and return is contradicting.

Reply:
Because the method is overridden in `NumberHelper` and `TextHelper` which do have return values, so keeping only `void` here makes pslam complain again :)